opendevin/runtime/e2b/sandbox.py

```diff
@@ -65,14 +61,6 @@ def _archive(self, host_src: str, recursive: bool = False):
                 tar.add(host_src, arcname=srcname)
         return tar_filename
 
-    # TODO: This won't work if we didn't wait for the background process to finish
```

Comment:
Re E2B: @enyst I think it doesn't really work _that_ well.

Reply:
That's fair! 

Cc: @mlejva Hopefully this doesn't break anything, and it simplifies things.

Reply:
Thanks for the pink @enyst . It should be good and it should simplify a lot of things I think. We have a new SDK update coming that will make things like background commands much easier if needed (the SDK is also much faster). 

It's already in beta release but we'll announce it on our socials ([Twitter](https://twitter.com/e2b_dev)) when prod ready.